selfdrive/can/tests/test_packer_toyota.py

```diff
@@ -8,6 +8,7 @@
   create_fcw_command, create_ui_command
 )
 from common.realtime import sec_since_boot
+from six.moves import range
```

Comment:
I don't like six, why do we need it here?

Reply:
Removing six.